### ISSUE TITLE
Implement capture attribute for input elements

### ISSUE THREAD

**Issue by macdonst** [phonegap/phonegap-plugin-pwa#9]
Make sure we can use the [capture](https://www.w3.org/TR/html-media-capture/) attribute to get audio, image or video data.

This could replace:
* [cordova-plugin-camera](https://github.com/apache/cordova-plugin-camera)
* [cordova-plugin-media-capture](https://github.com/apache/cordova-plugin-media-capture)
* and the recording section of [https://github.com/apache/cordova-plugin-media](https://github.com/apache/cordova-plugin-media)

Also, want to see how [getUserMedia/Stream](https://www.w3.org/TR/mediacapture-streams/) influences this.

**Comment by macdonst** [comment]
We've created a new plugin at: https://github.com/phonegap/phonegap-plugin-media-stream

That enables users to use getUserMedia on Android. For iOS it is more difficult but we've implemented the JS side of things so users can request a media stream and pass it to the Image Capture API.

**Comment by awebdeveloper** [comment]
This would be really awesome. If you can get capture working on android(using capture to open camera)

**Comment by macdonst** [comment]
@awebdeveloper have you looked at:

https://github.com/phonegap/phonegap-plugin-media-stream
https://github.com/phonegap/phonegap-plugin-image-capture

**Comment by awebdeveloper** [comment]
yup, but nothing beats `<input type="file" capture ........>` when i want to support upload + capture scenario

**Comment by HarelM** [comment]
@awebdeveloper Did you find something that can handle `<input type="file" capture ........>` for cordova?

**Comment by awebdeveloper** [comment]
apparently android doesn't support it. search on Cordova's jira you will find someone who has a fork that supports it
